Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.6.5
+FROM python:3.8
```

Comment:
[Docker Hub](https://hub.docker.com/_/python) appears to have a `3.8.6` version as opposed to simply `3.8` — considering you explicitly refer to `python-3.8.6` in `runtime.txt`, would it be possible/better to use this instead?

Reply:
The thing is, we cannot specify the `patch` version (the number after the second period) for Google App Engine, whereas we do need to specify it for Heroku. We can do either for Docker, so it will be inconsistent in any case, and I can't really argue for either side. To play devil's advocate, if there was a bug fixed in a more recent patch of Python, wouldn't it be easier to figure it out if the bug could be found either in the local environment or on Heroku? Although I think such a case would be very unlikely, I wouldn't suspect the Python patch version could be an issue if the bug could only be found in the production environment.

Reply:
@alex-shen1 But then you will stay in the team for a couple more years, so I specified the patch version too. I think my scenario was pretty unrealistic anyway.